internal/command/machine/restart.go

```diff
@@ -61,16 +60,29 @@ func newRestart() *cobra.Command {
 	return cmd
 }
 
-func runMachineRestart(ctx context.Context) error {
+func runMachineRestart(ctx context.Context) (err error) {
+
+	var (
+		args = flag.Args(ctx)
+	)
+
+	for _, machineID := range args {
+		if err = restart(ctx, machineID); err != nil {
+			return
+		}
+	}
+	return
+}
+
+func restart(ctx context.Context, machineID string) (err error) {
```

Comment:
it doesn't look like the return variable is being used (all the returns specify what they're returning rather than returning empty), so probably safer to remove it.

```suggestion
func restart(ctx context.Context, machineID string) error {
```